snmp/datadog_checks/snmp/data/default_profiles/checkpoint.yaml

```diff
@@ -175,6 +175,7 @@ metrics:
           0: false
           1: true
           2: reading_error
+          TODO: MAPPING NOT UPDATED - SYMBOL ENUMS NOT FOUND IN MIB DATABASE
```

Comment:
The mapping is from the description, comment to add for 7.49 https://github.com/DataDog/integrations-core/pull/15804